src/server/actions/org.actions.ts

```diff
@@ -87,6 +87,12 @@ export const updateOrgSettingsAction = orgAdminAction(async ({ orgSlug, name, de
     // TypeScript ensures orgFromContext.name is a string due to ActionContextOrgInfo type.
     const originalName = orgFromContext.name
 
+    // Check for duplicate name for existing organizations
+    const existingOrg = await db.selectFrom('org').select('id').where('name', '=', name).executeTakeFirst()
+    if (existingOrg && existingOrg.id !== orgFromContext.id) {
+        throw new ActionFailure({ name: 'Team name is already in use. Enter a unique name.' })
```

Comment:
are we calling orgs 'Teams' now?  Should this be "Organization name is already in use...", or maybe we leave it off and just say "Name is already in use"